### ISSUE TITLE
Writing a amiibo in shovel knight is causing a fatal error. In android

### ISSUE THREAD

**Issue by Xionic556** [citra-emu/citra#6615]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Canary 2915

### Description of Issue

Writing a amiibo in shovel knight is Causing a fatal error in android.

### Expected Behavior

The fatal error should not happen in android

### Reproduction Steps

Pick shovel knight amiibo
Press yes to register the amiibo to the game
Bug happen.

### Log File

[citra_log.txt.old.txt](https://github.com/citra-emu/citra/files/11758746/citra_log.txt.old.txt)

### System Configuration

SD 680
Adreno 610
Os 13

**Comment by FearlessTobi** [comment]
@german77 Can you please take a look?

**Comment by german77** [comment]
Are we sure is amiibos that cause this issue? log says this user has keys and handling the amiibo had no issues. 

This bug is probably found since citra wasn't able to write to amiibos before. Which is out of the scope of the NFC implementation. If this didn't crash before I could try getting the game and try to fix it. 
```
[  69.269270] Loader <Error> core/loader/loader.cpp:GetLoader:120: Failed to load file /nand/00000000000000000000000000000000/title/00040030/00009e02/content/00000000.app
[  69.269294] Service.NS <Warning> core/hle/service/apt/ns.cpp:LaunchTitle:17: Could not find .app for title 0x0004003000009e02
```

**Comment by german77** [comment]
It's also very likely that the game tried to open the amiibo settings applet to setup the amiibo owner. This crash is due the missing support for applets while a game is running on citra. Which is too out of my scope.

A workaround is to register the amiibo from the home menu first.

**Comment by Steveice10** [comment]
Yeah this is the amiibo setting applet trying to launch to register the amiibo. I still need to retest and look into it after some of the recent fixes I’ve made. For now it is possible to open amiibo settings through the home menu to register the amiibo if you want to try that. After registering the amiibo it should work fine in the game.
